src/main/kotlin/lotto/domain/Money.kt

```diff
@@ -0,0 +1,40 @@
+package lotto.domain
+
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+@JvmInline
+value class Money(
```

Comment:
from: https://github.com/next-step/kotlin-lotto/pull/1143#discussion_r2025513817
> One question I do have is To declare an inline class for the JVM backend, use the value modifier along with the @JvmInline annotation before the class declaration:
When is it used without JVM backend?

Sadly value classes without `@JvmInline` annotation are not supported yet... 
But this could be useful on [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) later on 🙂